packages/labs/src/components/select/multiSelect.tsx

```diff
@@ -176,8 +176,10 @@ export class MultiSelect<T> extends React.Component<IMultiSelectProps<T>, IMulti
     }
 
     private handleQueryChange = (e: React.FormEvent<HTMLInputElement>) => {
+        const { tagInputProps = {}, openOnKeyDown } = this.props;
         const query = e.currentTarget.value;
-        this.setState({ query, isOpen: !(query.length === 0 && this.props.openOnKeyDown) });
+        this.setState({ query, isOpen: !(query.length === 0 && openOnKeyDown) });
```

Comment:
This is a little tricky to parse mentally. I'd vote for `query.length > 0 || !openOnKeyDown)`.